xen.spec.in

```diff
@@ -666,11 +578,45 @@ done
 
 ############ all done now ############
 
+%post
+%if %with_systemd_presets
+%systemd_post xendomains.service
+%else
+if [ $1 == 1 ]; then
+  /bin/systemctl enable xendomains.service
+fi
+%endif
+
+%preun
+%if %with_systemd_presets
+%systemd_preun xendomains.service
+%else
+if [ $1 == 0 ]; then
+/bin/systemctl disable xendomains.service
+fi
+%endif
```

Comment:
`xendomains.service` is removed above, so this whole `%post` section should be too.